Automattic/wordpress-activitypub
# Header Images: fix cropping for users without the 'customize' capability

I think I found a Core bug here. There's absolutely no reason for the ajax request to include `wp_customize=on`, the logic is entirely handled in the `wp_ajax_crop_image()` handler.

[This commit](https://github.com/WordPress/wordpress-develop/commit/9fe59ced578169572f00d14a065aed22da165cab) appears to have moved the logic from the Customizer into a more general purpose context while [leaving the constraining `wp_customize=on` parameter](https://github.com/WordPress/wordpress-develop/commit/9fe59ced578169572f00d14a065aed22da165cab#diff-ce553f4ec6128c3384da17e519b3b0390203c996681ef1413ecf4b600fac5673R25).

I guess this could be a core patch

Fixes #846

## Proposed changes:

Make a custom controller but without `wp_customize=on`

### Other information:


## Testing instructions:

1. Be logged in as an author-level user (or any other user who can create posts but not edit others' posts)
2. Ensure that this user is ActivityPub-enabled
3. Go to profile and attempt to upload and crop an image. Any image uploaded or chosen from the library should work (the library will now only show the author user's own posts)
